TestHelpers.Tests/MockFileGetVersionInfoTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace System.IO.Abstractions.TestingHelpers.Tests
+{
+    using System.Diagnostics;
+    using System.Reflection;
+    using NUnit.Framework;
+
+    internal class MockFileGetVersionInfoTests
+    {
+        [Test]
+        public void MockFileVersionInfo_ToString_ShouldReturnMajorMinorBuildPrivate()
+        {
+            var version = new MockFileVersionInfo { FileMajorPart = 1, FileMinorPart = 2, FileBuildPart = 3, FilePrivatePart = 4 };
+            Assert.AreEqual("1.2.3.4", version.FileVersion);
+        }
+
+        [Test]
+        public void MockFileInfo_VersionInfo_ShouldBeAbleToSetMockVersionInfo()
+        {
+            var version = new MockFileVersionInfo { ProductName = "Testing MockFileVersionInfo" };
+            var fileData = new MockFileData("Demo text content") { VersionInfo = version };
+            Assert.AreSame(fileData.VersionInfo, version);
+        }
+
+        [Test]
+        public void MockFileInfo_VersionInfo_ShouldReadFromDll()
```

Comment:
We usually don't test the real file system, so I'd leave this out for consistency reasons.